doc/proxyfied-requests.md

```diff
@@ -160,6 +160,19 @@ httpRequest.getRawProxy("http://www.example.com/hello.zip")
     });
 ```
 
+### httpRequest.getBlobProxy
+
+This method can be used to retrieve data as a Blob. It returns a `Blob`.
```

Comment:
Like for the other occurrence: add a link to the Blob API:
```suggestion
This method can be used to retrieve data as a Blob. It returns a [`Blob`][blob].
```
+ at the bottom of the document:

```markdown
[blob]: https://developer.mozilla.org/en-US/docs/Web/API/Blob
```

Reply:
done